contracts/Remittance.sol

```diff
@@ -1,9 +1,9 @@
 pragma solidity >=0.4.21 <0.6.0;
 
 import "./Pausable.sol";
+import "./Ownable.sol";
```

Comment:
Looks unused.

Reply:
it's used in `onlyOwner`

Reply:
you're right, `Pausable` has imported `Ownable`